src/index.html

```diff
@@ -1,12 +1,357 @@
-<!doctype html>
-<html lang="en">
+<!DOCTYPE html>
+<html lang="en" class="page">
   <head>
-    <meta charset="UTF-8">
+    <meta charset="UTF-8" >
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Catalog</title>
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link
+      rel="preconnect"
+      href="https://fonts.gstatic.com" crossorigin="anonymous">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap" rel="stylesheet">
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
-  <body>
-    <h1>Catalog</h1>
+  <body class="page__content content">
+    <header class="header content__header">
+      <a href="#home" class="logo">
+        <img src="./images/logo.png" alt="moyoLogo">
+      </a>
+      <nav class="nav">
+        <ul class="nav__list">
+          <li class="nav__item">
+            <a href="#Apple" class="nav__link is-active"> Apple </a>
+          </li>
+
+          <li class="nav__item">
+            <a href="#Samsung" class="nav__link"> Samsung </a>
+          </li>
+
+          <li class="nav__item">
+            <a href="#Smartphones" class="nav__link"> Smartphones </a>
+          </li>
+
+          <li class="nav__item" data-qa="hover">
+            <a
+              href="#Laptops&Computers"
+              class="nav__link"
+              data-qa="nav-hover"
+            >
+              Laptops & Computers
+            </a>
+          </li>
+
+          <li class="nav__item">
+            <a href="#Gadgets" class="nav__link"> Gadgets </a>
+          </li>
+
+          <li class="nav__item">
+            <a href="#Tablets" class="nav__link"> Tablets </a>
+          </li>
+
+          <li class="nav__item">
+            <a href="#Photo" class="nav__link"> Photo </a>
+          </li>
+
+          <li class="nav__item">
+            <a href="#Video" class="nav__link"> Video </a>
+          </li>
+        </ul>
+      </nav>
+    </header>
+
+    <main class="cards content__cards">
+      <div class="cards__card card" data-qa="card">
+        <div class="card__img-wrap">
+          <img
+            class="card__img"
+            src="./images/imac.jpeg"
+            alt="card"
+          >
+        </div>
+
+        <h3 class="card__title">
+            APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+        </h3>
+
+        <p class="card__code">Product code: 195434</p>
+
+        <div class="card__rating">
+          <div class="card__stars stars stars--4">
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+          </div>
```

Comment:
Here an **ul** would be more semantically correct:
```suggestion
      <ul class="card__stars stars stars--4">
        <li class="stars__star"></li>
        <li class="stars__star"></li>
        <li class="stars__star"></li>
        <li class="stars__star"></li>
        <li class="stars__star"></li>
      </ul>
```